apps/idos-enclave/src/lib/enclave.js

```diff
@@ -125,14 +98,6 @@ export class Enclave {
         this.authorizedOrigins = [...new Set([...this.authorizedOrigins, this.parentOrigin])];
         this.store.set("enclave-authorized-origins", JSON.stringify(this.authorizedOrigins));
 
-        if (credentialId) {
-          this.store.set("credential-id", credentialId);
```

Comment:
We should start wiping this out from the store, too. Maybe I missed the part where we do that?